dagger-common/src/main/java/com/gotocompany/dagger/common/serde/proto/deserialization/ProtoDeserializer.java

```diff
@@ -3,11 +3,13 @@
 import com.google.protobuf.Descriptors;
```

Comment:
because we didnt overload the transformFromParquet() method, so that will not get affected, basically we are not enabling auto schema refresh for parquet

Reply:
This will be handled in code later